website/about.html

```diff
@@ -31,7 +31,7 @@
 			</div>
 			 </li>
         <li class="nav-item"><a class="nav-link text-white" href="faq.html">FAQ</a></li>
-        <li class="nav-item"><a class="nav-link text-white" href="memos.html">Memos</a> </li>
+        <li class="nav-item"><a class="nav-link text-white" href="memos.html">Memos, Talks, and Publications</a> </li>
```

Comment:
Personally I think this page title is too long. It makes the nav bar look very clunky.
<img width="835" alt="image" src="https://github.com/astrochart/astrochart.github.io/assets/4752917/0a7edca6-2305-4640-b8b1-61ca24192cc3">
Looking around at other collabs, most break up "Memos" and "Publications" into separate pages. I'm not sure I like this, especially because we currently only have the one publication. 
Maybe just labeling it "Papers" which is a more general term? or "Reports"?